setup.cfg

```
@@ -1,2 +1,2 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+long_description = README.md
```

To be honest, this configuration does not exist for setuptools.

The closest thing is the following:

```suggestion
long_description = file: README.md
```

See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html